IOPool/Common/bin/CollUtil.cc

```diff
@@ -112,7 +112,9 @@ namespace edm {
     std::cout << "UUID: " << fid.fid() << std::endl;
   }
 
-  static void preIndexIntoFilePrintEventLists(TFile*, FileFormatVersion const& fileFormatVersion, TTree *metaDataTree) {
+  static void preIndexIntoFilePrintEventLists(TFile *,
+                                              FileFormatVersion const &fileFormatVersion,
+                                              TTree *metaDataTree) {
```

Comment:
In this PR, I tried clang format again on IOPool. The namespace indentation looks OK now, but something still looks different from the prior times I ran this on Framework. For example, why is the symbol & next to fileFormatVersion above instead of adjacent to const?

Reply:
@wddgit , this is because of [a]. 
@fabiocos ,  @davidlange6 , @Dr15Jones should we make `DerivePointerAlignment: false` enforce a unique Alignment e.g. Left?
[a]
```
DerivePointerAlignment: true
DerivePointerAlignment (bool)
If true, analyze the formatted file for the most common alignment of & and *. 
Pointer and reference alignment styles are going to be updated according to
the preferences found in the file. PointerAlignment is then used only as fallback.
```